extensions/vertx-http/dev-console-runtime-spi/src/main/java/io/quarkus/devconsole/runtime/spi/DevConsolePostHandler.java

```diff
@@ -15,6 +15,7 @@ public abstract class DevConsolePostHandler implements Handler<RoutingContext> {
 
     @Override
     public void handle(RoutingContext event) {
+        event.request().resume();
```

Comment:
@cescoffier Please verify that it does not affect https://github.com/quarkusio/quarkus-quickstarts/tree/development/security-keycloak-authorization-quickstart, do `mvn quarkus-dev` and then access a DevConsole from a browser, since in #25470 `event.request().resume();` is only inside individual `if`/`else` branches